cmd/zc_traverser_s3.go

```diff
@@ -74,16 +74,29 @@ func (t *s3Traverser) traverse(processor objectProcessor, filters []objectFilter
 		// Otherwise, treat it as a directory.
 		// According to IsDirectorySyntactically, objects and folders can share names
 		if err == nil {
+			storedObject := newStoredObject(
```

Comment:
Well spotted that this had been missed.

BTW, I have some ideas on something we could do to avoid risk of missing such things during future maintenance.  But for simplicity, I'll leave those ideas out of this review. They can be discussed later.